mods/ra/rules/world.yaml

```diff
@@ -4,6 +4,7 @@
 	ActorMap:
 	ScreenMap:
 	Selection:
+	ControlGroups:
```

Comment:
An update rule would be nice.

Reply:
:thinking: Does this new trait need to be on the `^BaseWorld`? Or `World` is enough since control groups are not a thing in the editor?

Reply:
`World` would be the correct place as this is a strict in-game feature. Would give this refactor additional meaning.

Reply:
I guess I have to remove the `ControlGroup` field from the world actor then. And do trait look-ups every time?

Reply:
Oh no, that is far worse. It is probably the only reason why `Selection` is on `World`. I don't think it is used in the editor either.

Reply:
I pushed a fixup commit. I'll give the update rule a try as well.

Reply:
Our convention is to not add update rules for this kind of change, it should instead be described on the SDK update notes page.

Reply:
OK, should I drop the fixup commit 2f34786139ee4877276f40525b080458dbbd775d?

Reply:
Yes, drop it. Sorry for the misguiding suggestions.